nixos-generate

```diff
@@ -123,7 +135,7 @@ fi
 formatAttr=$(nix-instantiate "${nix_args[@]}" --eval --json -A config.formatAttr | jq -r .)
 filename=$(nix-instantiate "${nix_args[@]}" --eval --json -A config.filename | jq -r .)
 
-out=$(nix-build "${nix_args[@]}" --no-out-link -A "config.system.build.$formatAttr")
+out=$(nix-build "${nix_args[@]}" "${nix_build_args[@]}" -A "config.system.build.$formatAttr")
```

Comment:
if no `--no-out-link` is here, there will be a result symlink in the current directory. Or is this somehow fixed and I can't see it?

Reply:
Here: https://github.com/nix-community/nixos-generators/pull/66/files#diff-de075008d6bf41ab521680abe620e3b5R114-R116

If no out-link is set, no-out-link will be set

Reply:
ah, yeah, you are right.